composer.json

```diff
@@ -19,7 +19,7 @@
 			"composer/package-versions-deprecated": true
 		},
 		"platform": {
-			"php": "7.4"
+			"php": "8.0"
```

Comment:
If you run `composer update php` it will not update all dependencies but leave it to configured dependabot 🙈

Reply:
Good one!